{"uuid": "de7ad8e1-8b55-4cba-8d74-bcc2f7757cfc", "children": ["354bab6f-227e-4b43-8c30-9880055098a5"], "befores": [{"name": "pw_encryption", "status": "broken", "statusDetails": {"message": "TypeError: 'SubRequest' object is not iterable\n", "trace": "  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Administrator\\PycharmProjects\\test\\conftest.py\", line 23, in pw_encryption\n    pw = list(request)\n"}, "start": 1668157855809, "stop": 1668157855809}], "start": 1668157855809, "stop": 1668157855821}